src/fides/data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -217,4 +218,67 @@ privacy_notices:
+  internal_description: Targeted advertising" means displaying advertisements to a consumer where the advertisement is selected based on personal data obtained or inferred from that consumer's activities over time and across nonaffiliated Internet web sites or online applications to predict such consumer's preferences or interests.
+  regions:
+    - ca_ab
+    - ca_bc
+    - ca_mb
+    - ca_nb
+    - ca_nl
+    - ca_ns
+    - ca_on
+    - ca_pe
+    - ca_sk
+    - ca_nt
+    - ca_nu
+    - ca_yt
+  consent_mechanism: opt_out
+  data_uses:
+    - marketing.advertising.first_party.targeted
+    - marketing.advertising.third_party.targeted
+  enforcement_level: frontend
+  disabled: False
+  has_gpc_flag: True
+  displayed_in_privacy_center: True
+  displayed_in_overlay: True
+  displayed_in_api: False
+- name: Targeted Advertising and Marketing
+  notice_key: targeted_advertising_marketing
+  id: pri_c124c60f-b278-4ae6-a6f2-9b086b763tar
+  description: We may use some of your information and share it with partners for the purpose of marketing tracking and targeted advertising. You can learn more about what information is used for this purpose in our privacy notice.
+  internal_description: Advertising, targeting, or marketing cookies and services are technologies that tracks users' browsing behavior across websites. These technologies enable targeted advertising and marketing campaigns by collecting information about users' interests, preferences, and online activities. Marketing technologies typically require user consent under privacy regulations.
+  regions:
+    - ca_qc
+  consent_mechanism: opt_in
+  data_uses:
+    - marketing.advertising.first_party.targeted
+    - marketing.advertising.third_party.targeted
+  enforcement_level: frontend
+  disabled: False
+  has_gpc_flag: True
+  displayed_in_privacy_center: True
+  displayed_in_overlay: True
+  displayed_in_api: False
+- name: Functional and Essential
+  notice_key: functional_essential
+  id: pri_2ff6e1a1-2448-4db9-8a03-506810ed3fun
+  description: This website uses cookies and services to enable core website features, assist with privacy compliance, and provide a seamless user experience. These cookies and services are used to facilitate features such as navigation, remember user preferences, and ensure the security of the website. 
+  internal_description: Essential cookies and services, also known as strictly necessary, are technologies that are crucial for the proper functioning of a website. These technologies are typically exempt from requiring user consent under privacy regulations because they are essential for the website to deliver its basic functionalities and services or to comply with legal obligations such as tracking user consent.
+  regions:
+    - ca_qc
+  consent_mechanism: notice_only
+  data_uses:
+    - essential
+    - personalize
+    - improve.system
```

Comment:
ah ok the fideslang 2.0 migration - is this the only one that will need to be changed? https://github.com/ethyca/fides/issues/4008